node/src/gql/mutation/me/mentor-import.ts

```diff
@@ -0,0 +1,73 @@
+/*
+This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import {
+  GraphQLID,
+  GraphQLInputObjectType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import { MentorType } from 'gql/types/mentor';
+import { Mentor as MentorModel } from 'models';
+import { Mentor } from 'models/Mentor';
+import { Status } from 'models/Answer';
+import { SubjectUpdateInput, SubjectUpdateInputType } from './update-subject';
+import {
+  QuestionUpdateInput,
+  QuestionUpdateInputType,
+} from './update-question';
+
+export interface MentorImportJson {
+  subjects: SubjectUpdateInput[];
+  questions: QuestionUpdateInput[];
+  answers: AnswerUpdateInput[];
+}
+
+export const MentorImportJsonType = new GraphQLInputObjectType({
+  name: 'MentorImportJsonType',
+  fields: () => ({
+    subjects: { type: GraphQLList(SubjectUpdateInputType) },
+    questions: { type: GraphQLList(QuestionUpdateInputType) },
+    answers: { type: GraphQLList(AnswerUpdateInputType) },
+  }),
+});
+
+export interface AnswerUpdateInput {
+  question: QuestionUpdateInput;
+  transcript: string;
+  status: Status;
+}
+
+export const AnswerUpdateInputType = new GraphQLInputObjectType({
+  name: 'AnswerUpdateInputType',
+  fields: () => ({
+    question: { type: GraphQLNonNull(QuestionUpdateInputType) },
+    transcript: { type: GraphQLString },
```

Comment:
My concern was transcript might be null if the user never answered the question and importing/exporting that would cause an error if GraphQLNonNull was set. But if the user never answered a question, the answer would just not exist right?